docs/products/kafka/index.rst

```diff
@@ -14,10 +14,10 @@ Why Apache Kafka?
 
 Apache Kafka itself is technically a distributed log storage mechanism; in reality it is a best-in-class, highly-available data streaming solution. Oh, and it just happens to have an incredibly rich ecosystem of open source tooling that connects to and extends the existing platform.
 
-Aiven for Apache Kafka MirrorMaker2
-'''''''''''''''''''''''''''''''''''
+Aiven for Apache Kafka MirrorMaker 2
```

Comment:
shouldn't we have the ® also here?

Reply:
The ® would go on Kafka, and we've already done that in the top level heading and in the first paragraph.
I also didn't put an ® on the previous subtitle, which was just using `Apache Kafka`.

Also, the same applies to the next subtitle, Apache Kafka Connect.

So I *think* this is OK.